THEMES.md

```diff
@@ -517,6 +517,10 @@ Can be created as an extra.
 	- The image will be resized as large as possible so that it fits within this size and maintains its aspect ratio.  Use this instead of `size` when you don't know what kind of image you're using so it doesn't get grossly oversized on one axis (e.g. with a game's image metadata).
 * `origin` - type: NORMALIZED_PAIR.
 	- Where on the image `pos` refers to.  For example, an origin of `0.5 0.5` and a `pos` of `0.5 0.5` would place the image exactly in the middle of the screen.  If the "POSITION" and "SIZE" attributes are themable, "ORIGIN" is implied.
+* `rotation` - type: FLOAT.
```

Comment:
Ok, so I'm far from a Github Guru, but I understand it as follows:
Any change you now push to this branch, will automatically update this PR.
Now, if your remote for the retropie branch is called RP, you can do:
```
git fetch RP
git rebase RP/master GridView
```
Which should start a rebase activity, which will probably result in conflicts for those files you have currently merged explictly (screensaver/powersaver etc). Resolving those conflicts should be trivial, as these changes are identical. They are both in the same files, at the exact same locations though, so git will see them as overlapping edits, I think, (if not then the whole thing is painless anyways).
After you've resolved the conflicts, and added them to the commit, the rebase will finalize, resulting in a single commit with just your changes.

Good luck, let me know how that goes!